README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
I don't think it was a comment, but usually stupid webpages which had some JS part which replaced all quotes with smartquotes. But since then I associate smartquotes in code (including comments) with errors and more work and it feels kind of "dirty" (instead of your "beauty") and unprefoessional seeing them in source code parts. As if someone had programmed in MS word...

Beauty is in the eye of the beholder....